common/djangoapps/student/tests/test_authz.py

```diff
@@ -182,7 +182,7 @@ def setUp(self):
         self.global_admin = AdminFactory()
         self.creator = User.objects.create_user('testcreator', 'testcreator+courses@edx.org', 'foo')
         self.staff = User.objects.create_user('teststaff', 'teststaff+courses@edx.org', 'foo')
-        self.course_key = SlashSeparatedCourseKey('mitX', '101', 'test')
+        self.course_key = CourseLocator('mitX', '101', 'test')
```

Comment:
@ormsbee I agree with leaving SlashSeparatedCourseKey when it is specifically testing old mongo (or wants to test the 2 different types of keys). But a lot of these usages are just using SlashSeparatedCourseKey because of when the test was written, and they don't test split at all. If we are only going to test one modulestore, I think we should test the one we are primarily using....